site/src/testHelpers/entities.ts

```diff
@@ -966,6 +967,47 @@ export const MockWorkspaceAgent: TypesGen.WorkspaceAgent = {
 	],
 };
 
+export const MockWorkspaceChildAgent: TypesGen.WorkspaceAgent = {
+	apps: [MockWorkspaceApp],
```

Comment:
It might be too early to think about what this test payload should contain, but we could probably drop apps here if we want as they will often be zero.
```suggestion
	apps: [],
```